main.md

```diff
@@ -537,22 +541,21 @@ also issued a refresh token, this has the effect of DPoP-binding the refresh tok
 alone, which can improve the security posture even when protected resources are not 
 updated to support DPoP. 
 
-If a client receives a different `token_type` value than `DPoP` in the response, the
-access token protection provided by DPoP is not given. The client MUST discard the response in this
-case if this protection is deemed important for the security of the
-application and MAY continue as in a regular OAuth interaction otherwise.
+If the access token response contains a different `token_type` value than `DPoP` in the response, the
+access token protection provided by DPoP is not given.
+In this case, if this protection is deemed important for the security of the
+application, the client MUST discard the response and MAY continue as in a regular OAuth interaction otherwise.
 
-Refresh tokens issued to confidential clients (those having
-established authentication credentials with the authorization server) 
+Refresh tokens issued to confidential clients (see Section 2.3 of [@RFC6749]) 
 are not bound to the DPoP proof public key because they are already 
 sender-constrained with a different existing mechanism. The OAuth 2.0 Authorization 
 Framework [@!RFC6749] already requires that an authorization server bind
 refresh tokens to the client to which they were issued and that 
 confidential clients authenticate to the authorization server when 
 presenting a refresh token.  As a result, such refresh tokens
-are sender-constrained by way of the client ID and the associated 
+are sender-constrained by way of the `client_id` and the associated 
```

Comment:
I wanted to say client ID here not use a parameter name.

Reply:
Does "client identifier" work - according to oauth2?

Reply:
ok